register.js

```diff
@@ -133,17 +133,22 @@
 
 	// Returns an array of global event handlers names
 	// https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers
-	function getGlobalEventHandlersNames() {
-		var names = [];
-
-		if (!isNode) {
-			names =  Object.getOwnPropertyNames(HTMLElement.prototype).filter(
-				isGlobalEventHandler
-			);
-		}
-
-		return names;
-	}
+  function getGlobalEventHandlersNames() {
+    var names = [];
+    var isWorkerScope = typeof WorkerGlobalScope !== "undefined" &&
```

Comment:
at the top of this file we define the `isNode` so I think isWorker should be defined there as well.

Reply:
Good call.  Next commit has it.